CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.6.2] - 2017-04-19
```

Comment:
Should be 3.6.0 really, and the `[ ]` are there because there should be a URL footnote at the bottom of this file that matches this.

Also, generally we don't do version number bumps on PRs for these modules. Process for these is typically:

- Make a change and just add a line to the changelog
- When you're ready to do an npm release:
  - Add the version number above whatever lines are included
  - Rebuild and check there's nothing missing from the build output
  - npm publish
  - git commit & tag & push to master

Basically I'm saying you should do the version part of this after this PR is merged. We don't want to end up with versions in the changelog that aren't on npm.